nvflare/private/fed/app/utils.py

```diff
@@ -80,13 +80,13 @@ def create_admin_server(fl_server: FederatedServer, server_conf=None, args=None)
 
 
 def version_check():
-    if sys.version_info >= (3, 13):
+    if sys.version_info >= (3, 15):
         raise RuntimeError(
-            "Python versions 3.13 and above are not yet supported. Please use Python version between 3.8 and 3.12."
+            "Python versions 3.14 and above are not yet supported. Please use Python version between 3.9 and 3.14."
```

Comment:
Could you keep the upper limit version unchanged in this PR?
We’ll update our CI based on the newer Python version and adjust the upper limit afterward.